examples/slu/slurp/eval_utils/evaluation/metrics/distance.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Union
+
+from jiwer import wer
+from textdistance.algorithms.edit_based import levenshtein
+
+DISTANCE_OPTIONS = {"word", "char"}
+
+
+class Distance:
```

Comment:
Refactor all of this. Either move it into nemo and then import in your examples or put them all inside of your script inside examples. Dont do relative imports inside of examples.